src/cmd/top/top.c

```diff
@@ -290,12 +315,15 @@ int main (int argc, char *argv[])
         optparse_print_usage (opts);
         exit (1);
     }
+    color = colors_init (opts);
     if (!isatty (STDIN_FILENO))
         fatal (0, "stdin is not a terminal");
-    initialize_curses ();
+    initialize_curses (color);
 
     if (!(top = top_create (target, NULL, &error)))
         fatal (0, "%s", error.text);
+
+    top->color = color;
```

Comment:
good point, and that may mean it needs to be tweaked in #4842 too.

spending just a little time learning about the color initialization, I think doing `init_pair (..., -1, -1)` might set everything to use the current user's defaults.  So that might work.  Lemme give it a try.

Reply:
> spending just a little time learning about the color initialization, I think doing init_pair (..., -1, -1) might set everything to use the current user's defaults. So that might work. Lemme give it a try.

So this does appear to work.  The one difference is that b/c we have things like:

```
            wattron (sum->win, COLOR_PAIR(TOP_COLOR_GREEN) | A_BOLD);
```

the bolding of some stuff stays.  Do we consider "bolding" acceptable under "no color".  I think its ok.  Normal `top(1)` does some bolding.

Reply:
Yeah, bold is not a color I'd say. :-)